schema/deploy/util_functions/upsert_timestamp_columns.sql

```diff
@@ -116,21 +116,21 @@ $$ language plpgsql;
 
 comment on function cif_private.upsert_timestamp_columns(text, text, boolean, boolean, boolean, text, text)
   is $$
-  an internal function that adds the created/updated/deleted at/by columns, indices on fkeys,
+  an internal function that adds the created/updated/archived at/by columns, indices on fkeys,
   applies the _100_timestamps trigger,
-  applies the _050_immutable_deleted_records trigger
+  applies the _050_immutable_archived_records trigger
 
   example usage:
 
   create table some_schema.some_table (
     ...
   );
   select cif_private.upsert_timestamp_columns(
-  table_schema_name := 'some_schema',
-  table_name := 'some_table',
-  add_create := true,
-  add_update := true,
-  add_delete := true);
+  table_schema_name => 'some_schema',
+  table_name => 'some_table',
+  add_create => true,
```

Comment:
Just curiosity, what is the difference with the arrow here as opposed to `:=`?

Reply:
None, `:=` is just an older syntax: https://www.postgresql.org/docs/14/sql-syntax-calling-funcs.html